lib/spack/spack/installer.py

```diff
@@ -1154,11 +1157,12 @@ def build_process():
             pkg._total_time = time.time() - start_time
             build_time = pkg._total_time - pkg._fetch_time
 
-            tty.msg('{0} Successfully installed {1}'
-                    .format(pre, pkg_id),
-                    'Fetch: {0}.  Build: {1}.  Total: {2}.'
-                    .format(_hms(pkg._fetch_time), _hms(build_time),
-                            _hms(pkg._total_time)))
+            tty.debug('{0} Successfully installed {1}'
```

Comment:
I think this message should probably stay at message level.

Reply:
I changed it to reduce the output since the `[+] <intall location>` message will print to show it's been installed.

I'll change it back though.

Reply:
Changed.

Reply:
I was not demonstrating the greatest reading comprehension. Sorry about that, I thought this was the [+] message.

Reply:
Changed back to debug.